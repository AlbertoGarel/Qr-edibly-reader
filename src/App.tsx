import {StatusBar} from 'expo-status-bar';
import {StyleSheet, Text, View} from 'react-native';
// @ts-ignore
import {APP_NAME} from '@env'

export default function App() {
    return (
        <View style={styles.container}>
            <Text>Open up App.js to start working on your app!{APP_NAME}</Text>
            <StatusBar/>
        </View>
    );
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: '#fff',
        alignItems: 'center',
        justifyContent: 'center',
    },
});
