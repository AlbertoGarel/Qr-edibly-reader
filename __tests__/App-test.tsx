import renderer from 'react-test-renderer';
import App from "../src/App";
test('renders App.tsx correctly', ()=>{
    const tree = renderer.create("App").toJSON();
    expect(tree).toMatchSnapshot();
})