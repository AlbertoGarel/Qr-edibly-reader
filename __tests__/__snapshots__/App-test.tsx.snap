// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.tsx correctly 1`] = `"App"`;
